tests/commands/test_api_error.py

```diff
@@ -249,3 +249,60 @@ def test_record_tests(self):
 
         result = self.cli("record", "tests", "--session", self.session, "minitest", str(self.test_files_dir) + "/")
         self.assertEqual(result.exit_code, 0)
+
+    @responses.activate
+    @mock.patch.dict(os.environ, {"LAUNCHABLE_TOKEN": CliTestCase.launchable_token})
+    def test_all_workflow_when_server_down(self):
+        responses.add(
```

Comment:
Question: Don't we set a status code?

Reply:
We don't need it since the exception(ReadTimeout) is occurred in `client.request` method. Thus, we can't get a status code in this case even though we set it.

Reply:
OK, I could confirm it locally 👍